src/gui/vector/qgsattributesformproperties.cpp

```diff
@@ -662,6 +705,13 @@ QgsAttributeEditorElement *QgsAttributesFormProperties::createAttributeEditorWid
       break;
     }
 
+    case DnDTreeItemData::Action:
+    {
+      const QgsAction action { mLayer->actions()->action( itemData.name() )};
```

Comment:
```suggestion
      const QgsAction action = mLayer->actions()->action( itemData.name() );
```
I don't have a strong technical background but I'd like our code to be consistent and maybe discuss before starting integrating these init methods which I find less readable.

Reply:
> mainly minor code formatting comments.
> 
> What is you reasoning for using {}-initialisation? I am not sure if we discussed this in the past. I find this less readable and it is not coherent with the rest of the code base.
> If there is a good technical reason, then let's discuss/go for it. Otherwise, I'd prefer to stick to the current approach. But it is not my decision to take, just my 2 cents.

@3nids there have been some discussion about uniform initialization (aka brace-initialization), I believe we agreed upon not using it when initializing containers (because it may lead to unexpected results on some compilers), for the rest it's free to use. 

There are pros and cons by using braces, I personally like the fact that they prohibit narrowing and avoid the "most vexing parse" issue, this has helped me to find a few bugs with loss of precision.

Reply:
BTW it's a little late because brace initilizers are used everywhere in QGIS code.